uniflow-core/src/main/kotlin/io/uniflow/core/logger/UniFlowLogger.kt

```diff
@@ -24,22 +24,30 @@ import io.uniflow.core.flow.data.UIState
  * @author Arnaud Giuliani
  */
 object UniFlowLogger : Logger {
-    private val defaultLogger: Logger = SimpleMessageLogger()
-    private var logger = defaultLogger
+    private var _logger: Logger? = null
+    val logger: Logger
+        get() = _logger ?: error("No UniFlowLogger has been setup. Please setup a logger with UniFlowLogger.init(...)")
```

Comment:
Why do you prefer throwing instead of using a sensible default, e.g. `SimpleMessageLogger`, which is set using the `default` function (but it's _not_ the default?!)?

Reply:
yes, I can keep default one 👍